pages/c/_cluster/apps/charts.vue

```diff
@@ -313,6 +327,10 @@ export default {
         btnCb(false);
       }
     },
+
+    isPreRelease(version = '') {
+      return semver.prerelease(version) || version.includes('-rc');
```

Comment:
This isn't the right definition of prerelease.  See https://semver.org/ # 9 (but just contains hyphen doesn't work either, see 10).  The semver parser probably does this?

Reply:
This parser seems to have a pretty narrow definition of 'prerelease' which includes '-alpha' and '-beta' but not '-rc'; updated based off this definition:  

> ...a hyphen and a series of dot separated identifiers immediately following the patch version. Identifiers MUST comprise only ASCII alphanumerics and hyphens [0-9A-Za-z-]. Identifiers MUST NOT be empty.